docs/choosing_clusters.md

```diff
@@ -43,11 +43,37 @@ In the Docker For Mac preferences, click
 ### Cons
 
 - If Kubernetes breaks, it's easier to reset the whole thing than debug it.
+- Much more resource-intensive because it uses docker-shim as the container runtime.
 - Different defaults than a prod cluster and difficult to customize.
 - Not available on Linux.
 
 ---
 
+## Kind
+
+[Kind](https://kind.sigs.k8s.io/) runs Kubernetes inside a Docker container.
+
+The Kubernetes team uses Kind to test Kubernetes itself. But its fast startup
+time also makes it a good solution for local dev. Follow these instructions for
+setting up Kind with Tilt:
+
+[Kind Setup Instructions](https://github.com/windmilleng/kind-local)
+
+### Pros
+
+- Creating a new cluster is fast (~20 seconds). Deleting a cluster is even faster. 
+- Much more robust than Docker for Mac. Uses containerd instead of docker-shim. Short-lived clusters tend to be more reliable.
+- Supports a local image registry (with our [custom setup instructions](https://github.com/windmilleng/kind-local)).
+  Pushing images is fast. No fiddling with image registry auth credentials.
+- Can run in [most CI environments](https://github.com/kind-ci/examples) (TravisCI, CircleCI, etc)
+
+### Cons
+
+- The local registry setup is still new, and changing rapidly. You need to be using Tilt v0.12.0+
+- If Tilt can't find the registry, it will use the much slower `kind load` to load images.
```

Comment:
idk if this is a con worth expressing but: yeah cluster setup is relatively fast, but you'll probably be recreating your cluster more often than on d4m, which makes it FEEL slower sometimes

Reply:
hmmm...that sounds like more of an objection to a particular pattern of using Kind, than an objection to Kind itself.

I recall you suggested that during the cluster-api meeting (i.e., whether it was worthwhile to recreate the cluster each time), and they pushed back pretty hard on it.

let's listen to whether other users raise this tension, and how they frame it, and maybe that will help inform how to frame it here.

Reply:
that's fair!